### PR TITLE
Implemented Page404 with content.

### PR DESCRIPTION
**Pull request by TN4HPEL3**
@AnnaIstochnikova please check styles, use our variables and make it better with css.
- button functionality I will do later on.

### CODE REVIEW

**Comment by TN4HPEL3** [comment]
Did you check it?

**Comment by AnnaIstochnikova** [comment]
@TN4HPEL3 no, I haven't checked it yet. I'd like to prioritize other elements of the page first.

**Review by TN4HPEL3 — COMMENTED** [review]
Looks good.

File: src/pages/Page404/atoms/Page404Atom.jsx
Comment (AnnaIstochnikova) [inline src/pages/Page404/atoms/Page404Atom.jsx:1]: Do we need this atom?

**Review by AnnaIstochnikova — APPROVED** [review]

**Comment by AnnaIstochnikova** [comment]
@TN4HPEL3 could you please review this branch and merge it if everything looks good?

**Review by TN4HPEL3 — COMMENTED** [review]
All fine 👍

**Pull request merged**